functional_tests/step_helpers/footer_menu_helpers.py

```diff
@@ -0,0 +1,53 @@
+from behave.runner import Context
+from playwright.sync_api import Page
+
+
+def navigate_to_snippets(page: Page):
+    page.get_by_role("link", name="Snippets").click()
+    page.get_by_role("link", name="Footer menus").click()
+
+
+def create_new_footer_menu(page: Page):
+    """Click the 'Add footer menu' button from the Footer Menus snippet listing page."""
+    page.get_by_role("link", name="Add footer menu").click()
```

Comment:
I don't think these two functions add a lot of value since they're only used once and contain pretty self-explanatory code.

Reply:
Happy to remove them, I have no issues/ not too picky on this. @nehakerung